osu.Game/Overlays/RankingsOverlay.cs

```diff
@@ -22,18 +26,24 @@ public class RankingsOverlay : FullscreenOverlay
         protected readonly Bindable<Country> Country = new Bindable<Country>();
         protected readonly Bindable<RankingsScope> Scope = new Bindable<RankingsScope>();
         private readonly Bindable<RulesetInfo> ruleset = new Bindable<RulesetInfo>();
+        private readonly Bindable<APISpotlight> spotlight = new Bindable<APISpotlight>();
 
         private readonly BasicScrollContainer scrollFlow;
-        private readonly Container tableContainer;
+        private readonly Container contentContainer;
         private readonly DimmedLoadingLayer loading;
         private readonly Box background;
+        private readonly RankingsOverlayHeader header;
 
         private APIRequest lastRequest;
         private CancellationTokenSource cancellationToken;
+        private GetSpotlightsRequest spotlightsRequest;
```

Comment:
I'm not sure how I feel about this being all the way up here. I'd prefer it to be hidden somewhere below because now you have `lastRequest` and `spotlightsRequest` next to each other.